lvsfunc.py

```diff
@@ -828,7 +828,7 @@ def source(file: str,
         clip = core.imwri.Read(file)
         if not ref:
             clip = core.std.AssumeFPS(clip, fpsnum=ref.fps.numerator, fpsden=ref.fps.denominator)
-            clip = core.resize.Bicubic(clip, width=ref.width, height=ref.height, format=ref.format)
+            clip = core.resize.Bicubic(clip, width=ref.width, height=ref.height, format=ref.format, matrix_s=mvf.GetMatrix(ref))
```

Comment:
Note L829. If there is no `ref`, it obviously can't use properties from it. This is a flaw on my part, so I'll rewrite it at a later date. Use `fpsnum`/`fpsden`  and handle conversions after importing it for now until I get around to fixing it.